+tests/+lib/+costs/total_deaths_test.m

```diff
@@ -0,0 +1,21 @@
+% Load necessary data into workspace
+scr.init
+%% Get cost parameters
+
+% Input lethality vector
+l = [0.00022274,0.00022274,0.00022274,0.00022274,0.00022274,0.00221398,0.01351613,0.134212];
+
+% Run model
+% Call the lib.SIRV_model.
+M = lib.SIRV_model(1, ... The timestap DeltaT
+    'StartDate', '2020-10-09', ... The date for which the data should be loaded.
+    'Steps', 160 , ... The amount of steps to calculate beforehand.
+    'Method', "EulerForward", ... The numeric method to solve the ODE of the model.
+    'AgeGroups', 0:10:70, ... The way the age groups should be devided.
```

Comment:
You just input the default values of `Method`, `AgeGroups` and `ContactMatrixFile` here. 

You can leave them out if you want, but no harm is done to explicitly set them.